### PR TITLE
feat: see through lens

### LINKED ISSUE #1053: See the feed through the lens of someone else
Sometimes it's easy to get stuck in a bubble of only the people you're following, and you might be curious about what the feed looks like for someone else. 

We are proposing adding a feature where you can choose any profile to view the feed through.

The endpoint already exists for this, just need UI/FE development.

----Design to follow today----

### PR DESCRIPTION
**Pull request by sasicodes**
## What does this PR do?

Fixes #1053 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Enhancement (non-breaking small changes to existing functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How should this be tested?

- [ ] Test A
- [ ] Test B

## Checklist

- I haven't read the [contributing guide](https://github.com/lensterxyz/lenster/blob/main/CONTRIBUTING.md)
- My code doesn't follow the style guidelines of this project
- I haven't performed a self-review of my own code and corrected any misspellings
- I haven't commented my code, particularly in hard-to-understand areas
- I haven't checked if my PR needs changes to the documentation
- I haven't checked if my changes generate no new warnings
- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by bigint** [comment]
Good to merge @sasicodes?

**Comment by sasicodes** [comment]
> Good to merge @sasicodes?

nanum idhan ketutu irukaaa

**Comment by bigint** [comment]
@sasicodes With who?

File: apps/web/src/components/Home/SeeThroughLens.tsx [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+
+  return (
+    <Menu as="div" className="relative">
+      <Menu.Button className="rounded-md hover:bg-gray-300 p-1 hover:bg-opacity-20">
+        <span className="flex space-x-1 items-center text-sm">
+          <img
+            src={getAvatar(profile)}
+            loading="lazy"
+            className="bg-gray-200 w-5 h-5 rounded-full border dark:border-gray-700/80"
+            alt={profile?.handle}
```
Comment (bigint) [inline apps/web/src/components/Home/SeeThroughLens.tsx:59]: **suggestion:** add explicit height and width to prevent cls 🙏🏼

**Comment by bigint** [comment]
image

padding right and left looks different 🤔

File: apps/web/src/components/Home/SeeThroughLens.tsx [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+              </div>
+            ) : (
+              <>
+                {profiles.map((profile: Profile) => (
+                  <Menu.Item
+                    as="div"
+                    className={({ active }) =>
+                      clsx(
+                        { 'dropdown-active': active },
+                        'flex rounded-lg overflow-hidden gap-1 space-x-1 items-center cursor-pointer p-1'
```
Comment (bigint) [inline apps/web/src/components/Home/SeeThroughLens.tsx:121]: **question:** do we need flex and other classes here?

**Review by bigint — CHANGES_REQUESTED** [review]

File: apps/web/src/components/Home/Timeline/index.tsx [hunk 3]
```diff
@@ -39,12 +43,29 @@ const Timeline: FC = () => {
...
+      }
+    }
+    setRecommendedProfilesToSeeThrough(profiles?.slice(0, 5));
+  };
 
   const { data, loading, error, fetchMore } = useTimelineQuery({
-    variables: { request, reactionRequest, profileId }
+    variables: { request, reactionRequest, profileId },
+    onCompleted: (data) => {
+      const feedItems = data?.feed?.items as FeedItem[];
```
[the remark this answers is not in the document]
Comment (sasicodes) [reply apps/web/src/components/Home/Timeline/index.tsx:66]: yes

**Pull request merged**